modules/docs/arrow-docs/docs/docs/arrow/typeclasses/apply/README.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: docs
+title: Apply
+permalink: /docs/arrow/typeclasses/apply/
+redirect_from:
+  - /docs/typeclasses/apply/
+---
+
+## Apply
+
+`Apply` extends the `Functor` type class (which features the familiar `map` function) with a new function `ap`. The `ap` function is similar to `map` in that we are transforming a value in a context (a context being the `F` in `Kind<F, A>`; 
```

Comment:
I am currently working on the code and after that is done and build I finish the docs. 

I am aware that the text is incorrect and this will be corrected.